packages/fx-core/src/core/middleware/projectMigratorV3.ts

```diff
@@ -106,6 +106,7 @@ const Constants = {
     flag: "a+",
   },
   envFilePrefix: ".env.",
+  teamsManifestFileName: "manifest.json",
```

Comment:
As the manifest file name is not only used in migration period, move the constant to MetadataV3 in versionMetadata.ts is better.

Reply:
Updated, thx!